upgrade.sh

```diff
@@ -209,15 +209,16 @@ then
   if [ -d ros.d ];
   then
     echo "Restoring Bringup"
+    sudo mkdir -p /etc/ros/$ROSDISTRO/ros.d
```

Comment:
Before creating the directory, can you please check if the directory exists or not?  If the script is run on a robot that already has ROS configured this may crash if that directory already exists on-disk.

Reply:
:facepalm: yes, you are correct. I'm having a bad brain morning, apparently.